src/components/AppHeader/AppHeader.tsx

```diff
@@ -15,7 +15,11 @@ import ExitToAppIcon from "@material-ui/icons/ExitToApp";
 import Avatar from "@material-ui/core/Avatar";
 import { useStyles } from "./AppHeaderStyle";
 
-const AppHeader = () => {
+
+interface logoutType {
+  logout: () => void
```

Comment:
A good convention is to always prefix callbacks with `on`
```suggestion
  onLogout: () => void
```